tools/istio-iptables/pkg/validation/validator.go

```diff
@@ -23,6 +23,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/prometheus/common/log"
```

Comment:
wrong log package :slightly_smiling_face:

Reply:
Thanks! This is unrelated changes to this PR. Removed.